index.html

```diff
@@ -205,9 +220,12 @@ <h2 class="contact-form-title">Get In Touch</h2>
 
       </div>
       <div class="form-inputs">
-        <input type="text" placeholder="Full name">
-        <input type="text" placeholder="Email address">
-        <textarea name="message" id="messageID" cols="30" rows="10" placeholder="Enter text here"></textarea>
+        <div class="form-input-names">
+          <input name="name" type="text" placeholder="First name" maxlength="30" required>
+          <input name="name" type="text" placeholder="last name" maxlength="30" required>
+        </div>
+        <input name="email" type="email" placeholder="Email address" maxlength="30" required>
+        <textarea name="message" id="messageID" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
         <button type="submit" class="getintouchbtn">Get in touch</button>
```

Comment:
- [ ] Please, kindly add the `hover, active, and disabled` CSS pseudo-classes to all your buttons.

![button](https://user-images.githubusercontent.com/59397159/167255386-5577dd32-cc64-413d-adb8-15437d7dc977.png)